samples/snippets/tests/test_images.py

```diff
@@ -43,6 +48,39 @@ def test_disk():
     delete_disk(PROJECT, ZONE, test_disk_name)
 
 
+@pytest.fixture
+def test_snapshot(test_disk):
+    """
+    Make a snapshot that will be deleted when tests are done.
+    """
+    test_snap_name = "test-snap-" + uuid.uuid4().hex[:10]
+    snap = create_snapshot(PROJECT, test_disk.zone.rsplit('/')[-1], test_disk.name, test_snap_name)
+    yield snap
+    delete_snapshot(PROJECT, snap.name)
+
+
+@pytest.fixture
+def autodelete_image_name():
+    """
+    Provide a name for an image that will be deleted after the test is done.
+    """
+    test_img_name = "test-img-" + uuid.uuid4().hex[:10]
+    yield test_img_name
+
+    delete_image(PROJECT, test_img_name)
+
+
+@pytest.fixture()
+def autodelete_image(autodelete_image_name):
+    """
+    An image that will be deleted after the test is done.
+    """
+    src_img = get_image_from_family('debian-cloud', 'debian-11')
+    new_image = create_image_from_image(PROJECT, src_img.name, autodelete_image_name, 'debian-cloud',
+                                        storage_location='eu')
+    yield new_image
```

Comment:
The delete appears to be missing. Should there be a corresponding `delete` for the image created using `create_image_from_image`?

Reply:
This fixture uses `autodelete_image_name` fixture to get a name for the temporary image. The `autodelete_image_name` fixture does the deleting.